src/components/icons/svg/Ramp.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Path, SvgProps } from 'react-native-svg';
+import Svg from '../Svg';
+
+export function Ramp({ color, ...props }: SvgProps) {
```

Comment:
Probably want a different solution eventually so we can support new providers with remote config, but for now this is what we have.